src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/CleanUpRejectedFilesTask.java

```diff
@@ -53,13 +56,26 @@ public void run() {
                     .listBlobs(null, true, EnumSet.of(SNAPSHOTS), null, null)
                     .forEach(listItem -> {
                         try {
+                            final String zipFileName = FilenameUtils.getName(listItem.getUri().toString());
                             CloudBlockBlob blob = container.getBlockBlobReference(
-                                FilenameUtils.getName(listItem.getUri().toString())
+                                zipFileName
                             );
 
                             if (canBeDeleted(blob)) {
```

Comment:
do we need this `canBeDeleted` ?

Reply:
looks like no longer needed :). happy about it as it should then reduce this crazy level of nested ifs

Reply:
this will change business logic - canBeDeleted method checks if enough time passed after rejection.

Reply:
and I slightly refactored the code to reduce nesting

Reply:
It is required - rejected files need to stay there for a while before we delete them.